src/search/engines/etools/search.go

```diff
@@ -83,44 +83,59 @@ func (se Engine) Search(query string, opts options.Options, resChan chan result.
 
 	firstRequest := true
 
-	// Static params.
-	paramSafeSearch := safeSearchParamString(opts.SafeSearch)
+	// Constant params.
+	paramSafeSearchV := safeSearchValue(opts.SafeSearch)
 
 	for i := range opts.Pages.Max {
 		pageNum0 := i + opts.Pages.Start
 		ctx := colly.NewContext()
 		ctx.Put("page", strconv.Itoa(i))
 
-		var err error
 		// eTools requires a request for the first page.
-		if pageNum0 == 0 || firstRequest {
```

Comment:
pageNum0 is the first page, which is also firstRequest, so the first case is redundant since if that is true the second one is also true